gym_xiangqi/xiangqi_game.py

```diff
@@ -200,20 +217,15 @@ def run(self):
         """
         Run the game until terminating condition is achieved
         """
-        if self.on_init(self.agent_piece, self.enemy_piece):
-            clock = pygame.time.Clock()
-            self.running = True
+        clock = pygame.time.Clock()
+        self.running = True
 
         while self.running:
             clock.tick(FPS)
             for event in pygame.event.get():
                 self.on_event(event)
-
             self.render()
 
```

Comment:
Not displaying "gameover" at the end of the game anymore? Then, we can get rid of the method as well maybe?